src/test/java/com/rentals/UserServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package com.rentals;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.rentals.model.User;
+import com.rentals.repository.UserRepository;
+import com.rentals.service.UserService;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class UserServiceTest {
+
+	
+	
+	/*@RunWith(MockitoJUnitRunner.class)
```

Comment:
Please ensure that our code does not have `commented  lines`, it is for code maintainability